test/metabase/models/permissions_test.clj

```diff
@@ -407,9 +417,9 @@
                        #{"/db/1/schema/" "/db/2/schema/public/"}
                        #{"/db/1/schema/public/" "/db/2/schema/public/table/3/"}
                        #{"/db/1/schema/public/table/2/" "/db/3/schema/public/table/4/"}]]
-      (testing (pr-str (list 'perms/is-permissions-set? perms-set))
+      (testing (pr-str (list 'is-permissions-set? perms-set))
         (is (= true
-               (perms/is-permissions-set? perms-set))))))
+               (is-permissions-set? perms-set))))))
```

Comment:
Meta question: I noticed we tend to do `(is (= true (pred? x)))` instead of `(is (pred? x))`. Is there a reason why?